tests/api.c

```diff
@@ -80131,7 +80496,7 @@ static int test_wolfSSL_X509_load_crl_file(void)
 #ifdef WC_RSA_PSS
         ExpectIntEQ(wolfSSL_CertManagerVerify(store->cm,
             "certs/rsapss/server-rsapss-cert.pem", WOLFSSL_FILETYPE_PEM),
-            WC_NO_ERR_TRACE(CRL_CERT_REVOKED));
+            WC_NO_ERR_TRACE(ASN_NO_SIGNER_E));
 #endif
```

Comment:
So this one is a bit tricky. This test mixes use of X509_STORE and CertManager directly, which is not really designed that way for external use. The certs are added using `X509_LOOKUP_load_file()` but then instead of using associated X509_* interfaces to verify, the test reaches directly in to the CertManager and does a verify. This test does not form a chain to a self signed CA, previous behavior that was OK and so it proceeded to the CRL checks which failed. New behavior fails before CRL checks even happen as a chain cannot be made. I believe this is consistent with openssl error codes for the same inputs using X509_verify_cert().

Reply:
Ok. Since this is an older test and I don't believe that it was used for a specific case where the exact return code is expected.